2200094/1204최빈수.py

```diff
@@ -0,0 +1,29 @@
+
+# import sys
+# sys.stdin = open('input (4).txt')
+
+
+# 점수와 빈도수를 세트로 만들어 제일 큰 값만 출력
+
+
+
+T  = int(input())
+
+for t in range(1, T+1):
+    testcase = int(input())
+    scores = list(map(int, input().split())) # txt파일 불러와서 빨리 입력하는 법 기억이 안남 ㅜㅜ
+    result = 0  # 출력할 최빈수 변수 / 근데 딕셔너리로 만들거니깐 key 값을 할당해줄것이다.
+
+    dic = {} # (숫자 : 빈도수) 형태로 딕셔너리 만들기
+    for i in scores: # 점수를 하나씩 꺼내서
+        if i not in dic:     # 딕셔너리에 없으면(첫출현이라면)
+            dic[i] = 1    # 키생성, 값은 1로  // 딕셔너리[key] = value 형태로 삽입한다. 없으면 수정도 함
+        else:
+            dic[i] += 1   # 딕셔너리에 이미 생성되어있는 점수라면 값(빈도수) 1씩 더하기
+
+
+    count = 0  # 
+    for key, value in dic.items():
+        if count < value:
+            count = value # 할당시켜줌
+     # 그리고는 막혓습니다..
```

Comment:
```suggestion
    count = 0
    ans = 0
    for key, value in dic.items():
        if count < value:
            count = value
            ans = key
    print(f'#{t} {ans}')
```
거의 다 풀어주셨네요! 위와 같은 방법을 생각해볼 수 있을 것 같습니다.
이제 여기서 문제 조건 중 **최빈수가 여러 개라면 그 중 가장 큰 값을 출력한다** 라는 말이 있습니다! 테스트 케이스에서는 해당 경우가 주어지지 않았지만, 주어졌을 경우 어떻게 처리해야 할지도 생각해보시면 좋을 것 같습니다. 😄